CNV-16588: Document switching between multiple VM displays (#47319)
openshift/openshift-docs: modules/virt-switching-displays.adoc

// Module included in the following assemblies:
//
// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc

:_content-type: PROCEDURE
[id="virt-switching-displays_{context}"]
= Switching between VM displays

If your virtual machine (VM) has multiple displays, you can view each one by switching displays in the web console.

.Prerequisites

* The mediated device is configured in the `HyperConverged` custom resource and assigned to a VM.

.Procedure

> . In the OpenShift Virtualization console, click *Workloads* -> *Virtualization* from the side menu.
> . Click the *Virtual Machines* tab.

Reviewer: In 4.11 the Virtualization section is top menu (not under Workloads), the item is VirtualMachines
```suggestion
. In the OpenShift Virtualization console, click *Virtualization* from the side menu.
. Click the *VirtualMachines* tab.
```
![virtualmachinesMenuItem](https://user-images.githubusercontent.com/2181522/177036056-d31e5015-a6f3-4866-9ced-3c36c1e0b0af.png)

> . Select a Windows virtual machine to open the *Virtual Machine Overview* screen.

Reviewer: the title of the page is "Overview"
```suggestion
. Select a Windows virtual machine to open the *Overview* screen.
```
![Overview](https://user-images.githubusercontent.com/2181522/177036140-eaec371c-25be-496d-b024-2014a4595992.png)

. Click the *Console* tab.

> . Select the appropriate display from the *Send Key* drop down menu or by using the tabs below the drop down menu.

Reviewer: Can we add more information about what key combination to send for each display ?
```suggestion
. Select the appropriate display from the *Send Key* drop down menu or by using the tabs below the drop down menu, send key combination Alt+Ctl+1 for the first display and Alt+Ctl+2 for the second displat.
```

> . Select the appropriate display from the *Send Key* drop down menu or by using the tabs below the drop down menu. Use send key combination `Alt+Ctl+1` for the first display and `Alt+Ctl+2` for the second display.

Reviewer 2: According to the ISG, refer to the drop-down list by its label followed by the word "list".

Also, do not use location specific words such as "below" for accessibility requirements. Which tabs are you referring to below the drop-down menu? I didn't see anything that meets that description in the test environment.

I would suggest rewriting this step as something like this:

````
. Choose the appropriate key combination from the *Send Key* list:
.. To access the default VM display, select `Ctl + Alt+ 1`.
.. To access the vGPU display, select `Ctl + Alt + 2`.
````

Author: There's a new bug re: the tabs, so I'm going to remove that language for now as it doesn't look like it will be repaired in time for 4.11. We can then add the functionality once it's working as expected 👍 .
